Emby.Server.Implementations/Plugins/PluginManager.cs

```diff
@@ -94,9 +93,13 @@ public IEnumerable<Assembly> LoadAssemblies()
                 }
             }
 
+            var pluginType = typeof(IPlugin);
+            int dynamicallyAdded = 0;
+
             // Now load the assemblies..
-            foreach (var plugin in _plugins)
+            for (int i = 0; i <= _plugins.Count - dynamicallyAdded - 1; i++)
```

Comment:
```suggestion
            var originalPluginCount = _plugins.Count;
            for (int i = 0; i < originalPluginCount; i++)
```

And remove `dynamicallyAdded` counter

Reply:
done.